src/test/java/duke/util/DateTimeHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package duke.util;
+
+import duke.DukeException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DateTimeHandlerTest {
+
+    @Test
+    public void dateFormat_validInput_correctDateFormat() {
+        try {
+            Assertions.assertEquals("01 Sep 2020", DateTimeHandler.parseDate("1-9-2020"));
+            Assertions.assertEquals("01 Sep 2020", DateTimeHandler.parseDate("01-09-2020"));
+        } catch (DukeException e) {
+            Assertions.fail();
+        }
+    }
+
+    @Test
+    public void dateFormat_invalidInput_dukeExceptionThrown() {
```

Comment:
Perhaps can explain more clearly what does it mean by having an invalidInput?

Reply:
Changed to dateFormat_invalidFormat_dukeExceptionThrown().